Sources/SpeziOpenAI/MessageView.swift

```diff
@@ -38,7 +39,10 @@ public struct MessageView: View {
     
     
     public var body: some View {
-        if chat.role != .system || (chat.role == .system && !hideSystemMessages), let content = chat.content {
+        if chat.role != .system ||
+           (chat.role == .system && !hideSystemMessages) ||
+           (chat.role == .function && !hideFunctionMessages),
+           let content = chat.content {
```

Comment:
There is an issue with the current implementation: `chat.role != .system` ensures that `.function` messages are always displayed. By using a set, we can greatly simplify this statement, e.g. like this:
```suggestion
       if !hideMessagesWithRoles.contains(chat.role), let content = chat.content {
```